getting-started/proc-configuring-oidc-with-keycloak.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-configuring-oidc-with-keycloak_{context}"]
+
+= Configuring OIDC with Keycloak in {product-short}
```

Comment:
Noted that this is commented out and intentionally not showing up in the preview

Reply:
Yes